backend/backend/settings.py

```
@@ -26,8 +26,10 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", "unsafe-default-key")
-RECAPTCHA_PUBLIC_KEY = os.environ.get("RECAPTCHA_PUBLIC_KEY")
-DRF_RECAPTCHA_SECRET_KEY = os.environ.get("RECAPTCHA_PRIVATE_KEY")
+RECAPTCHA_PUBLIC_KEY = os.environ.get("RECAPTCHA_PUBLIC_KEY", "unsafe-recaptcha-key")
+DRF_RECAPTCHA_SECRET_KEY = os.environ.get(
+    "RECAPTCHA_PRIVATE_KEY", "unsafe-recaptcha-key"
+)
```

Defaulting RECAPTCHA_PUBLIC_KEY and DRF_RECAPTCHA_SECRET_KEY to fixed "unsafe-" values in settings may leak insecure defaults into deployed environments and mask misconfiguration. This can allow bypass or unexpected behavior in production.
Do not provide secret defaults; require explicit environment values or fail fast:
```
RECAPTCHA_PUBLIC_KEY = os.environ["RECAPTCHA_PUBLIC_KEY"]
DRF_RECAPTCHA_SECRET_KEY = os.environ["RECAPTCHA_PRIVATE_KEY"]
```
Or at minimum log/raise an error on missing values in non-DEBUG mode.

---

Done.